config/dev.config.template

```diff
@@ -18,3 +18,9 @@ DB_HOSTNAME=127.0.0.1
 DB_PORT=3306
 DB_USERNAME=root
 DB_PASSWORD=
+
+#REDIS
+REDIS_HOST=127.0.0.1
+REDIS_PORT=6379
+REDIS_PATH=
+REDIS_URL=
```

Comment:
Is it necessary to set either or both of these parameters set at run-time? If so, there should be some sensible defaults.